deeppavlov/download.py

```diff
@@ -76,7 +76,12 @@ def get_configs_downloads(config: Optional[Union[str, Path, dict]]=None) -> Dict
 
 
 def check_md5(url: str, dest_paths: List[Path]) -> bool:
-    r = requests.get(url + '.md5')
+    if "?" in url:
+        url_md5 = url.replace("?", '.md5?')
```

Comment:
You can use https://docs.python.org/3.6/library/urllib.parse.html#urllib.parse.urlparse
and https://docs.python.org/3.6/library/collections.html#collections.somenamedtuple._replace

Reply:
add a def to modify an URL